deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -423,6 +424,26 @@ async def create_new_job(
                 job_type=service,
             )
 
+            # NOTE this happens after the job is already created to have access to the job id, hopefully this doesn't cause any issues with the job starting before the file is uploaded to s3
+            # Versions less than 0.28.0 should not be in the dropdown as an option, this is just an extra check for versions greater than 0.28.0
```

Comment:
Is this saying that we have to deprecate all versions before 0.28.0 for new analyses or just interactive analysis? People might still be using older versions for both so I'm not sure that we can do that for either right now